enhancements/network/service-egress-traffic-steering.md

```diff
@@ -287,7 +316,16 @@ type EgressService struct {
 
 // EgressServiceSpec defines the desired state of EgressService
 type EgressServiceSpec struct {
-  // Allows limiting the nodes that can be selected to handle the service's traffic.
+  // Determines the source IP of egress traffic originating from the pods backing the Service.
+  // When `true` the source IP is set to its LoadBalancer ingress IP.
+  // When `false` the source IP is set according to the interface of the Network,
+  // leveraging the masquerade rules that are already in place.
+  // Typically these rules specify SNAT to the IP of the outgoing interface, 
+  // which means the packet will typically leave with the IP of the node when the field is `false`.
+  // +optional
+  SNATToLoadBalancerIP bool `json:"snatToLoadBalancerIP,omitempty"`
```

Comment:
I don't think it is advised to use bool in the API: https://github.com/openshift/crd-schema-checker/blob/master/pkg/manifestcomparators/comp_no_bools.go#L21

Reply:
done, changed to SourceIPMode